galata/test/jupyterlab/settings.test.ts

```diff
@@ -198,3 +198,35 @@ test.describe('change font-size', () => {
     expect(computedStyle.fontSize).toEqual(`${fontSize - 1}px`);
   });
 });
+
+test('Check codemirror settings can all be set at the same time.', async ({
+  page
+}) => {
+  await page.evaluate(async () => {
+    await window.jupyterapp.commands.execute('settingeditor:open', {
+      query: 'CodeMirror'
+    });
+  });
+
+  expect(
+    await page.locator('.jp-PluginList .jp-FilterBox input').inputValue()
+  ).toEqual('CodeMirror');
+
```

Comment:
```suggestion
  expect(
    await page.locator('.jp-PluginList .jp-FilterBox input').inputValue()
  ).toEqual('CodeMirror');

```

this is tested separately with 'Command Palette', let's simplify.